### DISCUSSION TITLE
Decide what goes in the CONTRIBUTING guidelines and what in the wiki of qiskit.org

### DISCUSSION THREAD

**Discussion by delapuente** [Qiskit/qiskit.org#2386] [General]
The CONTRIBUTING.md should focus on the external contributor. Perhaps the things more related to the core dev team can go in the Wiki.

**Comment by y4izus** [comment]
I've added some comments [here](https://github.com/Qiskit/qiskit.org/pull/1035) to separate content

**Comment by techtolentino** [comment]
Sections / content that should go into the Wiki (taken from Va's Box note)

- Welcome to the Qiskit Community
  - Themes / sub-themes
  - Open Source Contributions *
  - GitHub *
  - Learning Quantum
- Values and Principles

* = duplicate?

--

Potential additional wiki content could include
- How we use Carbon, or to what extent we use Carbon
- Anything team specific with regards to using Nuxt
  -  i.e., class-based components
  -  typed annotations
- Class naming conventions

**Comment by delapuente** [comment]
I've performed a basic split between [CONTRIBUTING.md](https://github.com/Qiskit/qiskit.org/blob/master/CONTRIBUTING.md) and the [Wiki](https://github.com/Qiskit/qiskit.org/wiki). I leave the rest to you. I've also opened some extra issues with the things I need still remain to do. You can find them under the [`type: documentation` tag](https://github.com/Qiskit/qiskit.org/issues?q=is%3Aopen+is%3Aissue+label%3A%22type%3A+documentation%22).

**Comment by delapuente** [comment]
Also, @techtolentino @y4izus @vabarbosa feel free to open new documentation issues with those aspects you think deserve written explanations like those suggested by Randy: use of Carbon, class-based components, naming conventions...
